src/clr-angular/popover/dropdown/dropdown-menu.ts

```diff
@@ -57,6 +57,7 @@ export class ClrDropdownMenu extends AbstractPopover implements AfterContentInit
       this.popoverPoint = Point.LEFT_TOP;
     }
     this.popoverOptions.allowMultipleOpen = true;
+    this.popoverOptions.ignoreGlobalESCListener = true;
```

Comment:
So drop down bypasses the ESC listener in AbstractPopover and implements its own event listeners. This makes sense to me. 👍🏽